spring-boot-parent/pom.xml

```diff
@@ -44,14 +89,43 @@
                         <mainClass>${start-class}</mainClass>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>com.google.cloud.tools</groupId>
+                    <artifactId>jib-maven-plugin</artifactId>
+                    <configuration>
+                        <from>
+                            <image>${docker.base.image}</image>
+                            <platforms>
+                                <platform>
+                                    <os>linux</os>
+                                <architecture>${docker.base.arch}</architecture>
+                                </platform>
+                            </platforms>
+                        </from>
+                        <to>
+                            <image>${image.path}</image>
+                        </to>
+                        <container>
+                            <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+                        </container>
+                        <allowInsecureRegistries>true</allowInsecureRegistries>
```

Comment:
to powoduje ze nie  trzeba sie uwierzytelniac?

Reply:
Tak. To dotyczy lokalnego rejestru. Nasze  obrazy wchodzą, wychodzą do lokalnego rejestru.